index.html

```diff
@@ -1222,19 +1222,36 @@ <h1>Common Deployment Patterns</h1>
     </p>
 
     <section id="telemetry">
-      <h3>Telemetry</h3>    
-      <section class="ednote" title="TODO">
-      <h3>TODO</h3>
-      Create a description and a diagram for a telemetry scenario.
-      </section>
+      <h3>Telemetry</h3>  
+      <p>
+        Telemetry is the monitoring of remove devices, which implies automatic transmission to, and processing
+        of measurements and other data on a <a>Consumer</a>. 
+        Data may be transported over various bearers, both wireless and wired.
```

Comment:
bearer is not defined in the terminology

Reply:
@ege We could use "communication channels" as an alternative wording.

Reply:
I think that would be better :)